src/client/components/ui/Card/Flipping/FlippingCard.tsx

```diff
@@ -43,7 +43,6 @@ export function FlippingCard(props: FlippingCardProps) {
 		}
 	}, [category]);
 
-
 	useEffect(function setFocusOnFlip() {
 		if (isCardFlipped) {
 			const currentItem = cardFlipRef.current as HTMLDivElement;
```

Comment:
Malheureusement, il y a une raison pour laquelle la ref est de type `HTMLDivElement | null`
```suggestion
			const currentItem = cardFlipRef.current;
			const firstElement = currentItem?.getElementsByTagName('button')[0];
			firstElement?.focus();
```